downstream/modules/platform/ref-controller-job-slice-execution-behavior.adoc

```diff
@@ -0,0 +1,20 @@
+[id="controller-job-slice-execution-behavior"]
+
+= Job slice execution behavior
+
+When jobs are sliced, they can run on any node and some may not run at the same time (insufficient capacity in the system, for example). 
```

Comment:
When jobs are sliced, they can run on any node. Insufficient capacity in the system can cause some to run at a different time.